tests/wpt/metadata/html/semantics/embedded-content/media-elements/interfaces/HTMLElement/HTMLMediaElement/crossOrigin.html.ini

```diff
@@ -1,31 +1,31 @@
 [crossOrigin.html]
   [HTMLMediaElement.crossOrigin, setting to invalid value]
-    expected: FAIL
+    expected: PASS
```

Comment:
If these subtests all pass, you can actually delete this file. By default, all tests are expected to PASS, so having no INI file means exactly that.

Reply:
Oh neat, thanks.